api/node/routes.go

```diff
@@ -56,6 +58,7 @@ func Routes(router *gin.RouterGroup) {
 	router.GET("/address", Address)
 	router.GET("/heartbeatstatus", HeartbeatStatus)
 	router.GET("/statistics", Statistics)
+	router.GET("/details", StatusMetrics)
```

Comment:
instead of details you can call it status and remove the current status which shows the node up or not.

Reply:
done